composer.json

```diff
@@ -0,0 +1,29 @@
+{
+	"name": "php-solr/php-solr",
+	"type": "php-ext",
+	"license": "PHP-3.01",
+	"description": "Apache Solr extension",
+	"require": {
+		"php": ">= 7.4,<8.5",
```

Comment:
>= 8.1

Reply:
@omars44 I agree that we should aim to sync with PHP on the versions. At least for this new pie stuff. Perhaps not remove support for old versions in code unless we need to.

I will rebase the packaging PR when this is merged.